cogs/utils/constants.py

```diff
@@ -2,13 +2,17 @@
 
 from discord import Color
 
+DEV_MODE = True
+
+owner_id = 688275913535914014
```

Comment:
What is this?

Reply:
My ID? I Changed it to a list of us 3 soon